pkg/gadgets/run/tracer/tracer.go

```diff
@@ -360,8 +407,27 @@ func (t *Tracer) processEventFunc(gadgetCtx gadgets.GadgetContext) func(data []b
 		typ   endpointType
 	}
 
+	type wasmColumnDef struct {
+		setter func(*types.Event, string)
+		name   string
+		start  uint32
+		size   uint32
+	}
```

Comment:
I'm wondering if we could use an approach similar to the enumeration support: some "setters" are created, then the tracer only calls them. I tried it on https://github.com/inspektor-gadget/inspektor-gadget/commit/58e5e7b123cac3c5e299573595b8a48f59a08140.